lib/call-session.js

```diff
@@ -698,7 +693,7 @@ Duration=${payload.duration} `
 
       /* if this is a re-invite from the FS to change media anchoring, avoid sending the reinvite out */
       let sdp;
-      if (isReleasingMedia && !this.callerIsUsingSrtp) {
```

Comment:
Insteead of removing this line, please comment it out with a comment above, something like
```
// HL 2024-11-13: previously forwarded re-invites to webrtc clients but further testing has shown to be unnecessary
```

Reply:
fixed